persistent-state.js

```diff
@@ -178,10 +195,10 @@ class PersistentState extends HTMLElement {
       let config = this.storage.customElementConfigs[tagName]
       elem.addEventListener(config.changeEvent, (e) => {
         let value = config.onChange(e);
-        this.storage.set(key, value, tagName, this._storageId)
+        this.storage.set(key, value, this.type, this._storageId, config.isJSON)
       });
       this._resetCallbacks.push(() => {
-        this.storage.reset(tagName, key, this._storageId)
+        this.storage.reset(this.type, key, this._storageId)
```

Comment:
This was a bug that @JHStoops found. `this.type` represents the storage type, not the tag type (which is what I thought when I published it originally).